public/schema/README.md

```diff
@@ -0,0 +1,9 @@
+# Splits I/O JSON Schema
+The Splits I/O JSON Schema is a standard way of arranging run information in JSON.
+
+[**Documentation**][1]
+
+[**Raw JSON Schema**][2]
+
+[1]: http://lbovet.github.io/docson/index.html#https://raw.githubusercontent.com/glacials/splits-io/master/public/schema/run_v1.0.0.json
```

Comment:
This link won't work before merging, but you can use http://lbovet.github.io/docson/index.html#https://raw.githubusercontent.com/glacials/splits-io/standard_format/public/schema/run_v1.0.0.json to see what it looks like.